ci/test/01_base_install.sh

```diff
@@ -48,12 +48,11 @@ if [ -n "$PIP_PACKAGES" ]; then
 fi
 
 if [[ ${USE_MEMORY_SANITIZER} == "true" ]]; then
-  update-alternatives --install /usr/bin/clang++ clang++ "$(which clang++-12)" 100
-  update-alternatives --install /usr/bin/clang clang "$(which clang-12)" 100
-  mkdir -p "${BASE_SCRATCH_DIR}"/msan/build/
-  git clone --depth=1 https://github.com/llvm/llvm-project -b llvmorg-12.0.0 "${BASE_SCRATCH_DIR}"/msan/llvm-project
-  cd "${BASE_SCRATCH_DIR}"/msan/build/ && cmake -DLLVM_ENABLE_PROJECTS='libcxx;libcxxabi' -DCMAKE_BUILD_TYPE=Release -DLLVM_USE_SANITIZER=MemoryWithOrigins -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DLLVM_TARGETS_TO_BUILD=X86 ../llvm-project/llvm/
-  cd "${BASE_SCRATCH_DIR}"/msan/build/ && make "$MAKEJOBS" cxx
+  update-alternatives --install /usr/bin/clang++ clang++ "$(which clang++-16)" 100
+  update-alternatives --install /usr/bin/clang clang "$(which clang-16)" 100
+  git clone --depth=1 https://github.com/llvm/llvm-project -b llvmorg-16.0.0 "${BASE_SCRATCH_DIR}"/msan/llvm-project
```

Comment:
Could also bump this to 16.0.1 now https://github.com/llvm/llvm-project/compare/llvmorg-16.0.0...llvmorg-16.0.1.

Reply:
Moved to 16.0.1.